crates/provider-blobstore-azure/Cargo.toml

```diff
@@ -0,0 +1,39 @@
+[package]
+name = "wasmcloud-provider-blobstore-azure"
+version = "0.1.0"
+description = """
+Blobstore for wasmCloud, leveraging Azure Blobstore. This package provides a capability provider that satisfies the 'wasi:blobstore' world.
+"""
+
+authors.workspace = true
+categories.workspace = true
+edition.workspace = true
+license.workspace = true
+repository.workspace = true
+
+# See more keys and their definitions at https://doc.rust-lang.org/cargo/reference/manifest.html
+
+[dependencies]
+anyhow = { workspace = true }
+async-nats = { workspace = true }
+async-trait = { workspace = true }
+tokio = { workspace = true }
+path-clean = { workspace = true }
+serde = { workspace = true }
+tokio-stream = { workspace = true, features = ["fs"] }
+tokio-util = { workspace = true, features = ["io"] }
+tracing = { workspace = true }
+wasmcloud-provider-sdk = { workspace = true, features = ["otel"] }
+wrpc-interface-blobstore = { workspace = true }
+wrpc-transport = { workspace = true }
+base64 = { workspace = true }
+bytes = { workspace = true }
+serde_json = { workspace = true }
+simple_env_load = { workspace = true }
+time = { workspace = true }
+futures = { workspace = true }
+
+# As these crates are very specific to this provider, they are imported independently from the workspace
+azure_storage_blobs = { version = "0.19" }
+azure_storage = { version = "0.19" }
+azure_core = { version = "0.19" }
```

Comment:
nit; I would have preferred to have these in the workspace deps for easier tracking of *all* dependencies of the project in a single location

Reply:
Even if they are super specific to the individual provider? I don't mind either way really, I just thought it was weird to have a dep in a place where it's only relevant to one thing. I'll move it since we do it with other providers like s3